modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/dependencies_rbac.bicep

```diff
@@ -0,0 +1,16 @@
+targetScope = 'subscription'
```

Comment:
I'd suggest to update the documentation to the following proposal #2251.

@fabmas & @eriqua what would be your thoughts? @eriqua I'd appreciate if we get rid of the file and move the dependency. BUT if we do, we should also change it for the Compute-Disk-Import Test (in a separate PR).

Reply:
@MrMCake for compute-disk-import and compute-images we cannot get rid of the additional rbac module unfortunately, since we'd need the roleassignment to be done before other dependencies are deployed at rg scope (i.e. deployment scripts to bake a vhd file, which is not needed in this scenario)